packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -47,6 +47,11 @@ const cli = {
         describe: 'A flag to disable the selenium docker service.',
         default: false,
       },
+      disableSnapshotMismatch: {
```

Comment:
We use the word "screenshot" instead of "snapshot" in `updateScreenshots`. I think we should be consistent and also use "screenshot" here. I also think we should use the word "ignore" instead of "disable" since we're not disabling the comparisons...we are only ignoring.

Name suggestions:
`ignoreScreenshotMistmatch`
`ignoreScreenshotComparison`

Reply:
+1 @benbcai Either of your suggested names would work for me.

Reply:
Renamed to `ignoreScreenshotMistmatch`, also updated description to say screenshot
https://github.com/cerner/terra-toolkit/pull/655/commits/8571fc4e7f0de6b5aa3178eb5c70084c3fa80539#diff-b821c3a083e050c54034354280dac343f0b8d9ad66c460cad80a99ec7ebea640R50

Reply:
mistmatch isn't a word - let's update this to `ignoreScreenshotMismatch`

Reply:
Sorry, it was my typo.

Reply:
eh, we can also consider this a lesson of not blindly applying a change :)

Reply:
yep, renamed here https://github.com/cerner/terra-toolkit/pull/655/commits/9b9e06d9786ebf0471e3f3e0f7c2df80639a004c#diff-5121be21da689ee0a1bd5a529bbd578dc2bde92ce93b497f118e58290be63dc1R6